src/annotator/anchoring/text-range.ts

```diff
@@ -314,4 +316,12 @@ export class TextRange {
       new TextPosition(root, end)
     );
   }
+
+  /**
+   * Return a new Range representing `range` trimmed of any leading or trailing
+   * whitespace
+   */
+  static trimmedRange(range: Range): Range {
+    return trimRange(TextRange.fromRange(range).toRange());
```

Comment:
Ah. I was originally thinking to wrap just the `TextRange.fromRange(range).toRange()` part in a static method to explain why this apparently redundant thing was being done.